plugins/webp-uploads/hooks.php

```diff
@@ -747,13 +747,84 @@ function webp_uploads_render_generator(): void {
 }
 add_action( 'wp_head', 'webp_uploads_render_generator' );
 
+/**
+ * Process a block's content to handle background images for specific block types.
+ *
+ * This function targets blocks like Cover and Group that may use background images,
+ * converting them to modern image formats when appropriate.
+ *
+ * @since n.e.x.t
```

Comment:
We can add more specific typing for this arg:

```suggestion
 * @since n.e.x.t
 *
 * @phpstan-param array{
 *                    blockName: string|null,
 *                    attrs: array{
 *                        id?: positive-int,
 *                        url?: string,
 *                        style?: array{
 *                            background?: array{
 *                                backgroundImage?: string
 *                            }
 *                        }
 *                    }
 *                } $block
```